tests/selectedtests/task_mappings/test_create_task_mappings.py

```diff
@@ -86,11 +90,14 @@ def test_module_source_files_included(
         only_version_analyzed = evg_api_mock.versions_by_project.return_value[1]
 
         associated_module_mock.return_value = None
-        module_changed_mock.return_value = {"module_file"}
-        filtered_mock.return_value = {f"file{i}" for i in range(2)}
+        module_changed_mock.return_value = {ChangedFile("module_file", "my_module_repo")}
+        filtered_files_mock.return_value = {
```

Comment:
Why change all these assignments from using comprehensions to creating each instance manually? The comprehensions tend to be more flexible (e.g. if we wanted to create 5 values instead of 2 it is a 1 character change). Also if we change how the value is represented (like it this case), we only have to change how it is created in one place instead of changing each instance.

Reply:
I'm happy to change it back, I just find it easier to read hard coded variables rather than ones created by loops. So if there were fewer than three items being created, I removed the loop.

Reply:
We are doing it enough in this file that it might make sense to just have a helper function that creates some number of `ChangedFile`s. Then just have all the instances you changed just call the function. That way if we make changes in the future, we just have 1 place to change.